### ISSUE TITLE
Projects appear to be duplicates?

### ISSUE THREAD

**Issue by JGelfand** [FRCTeam159/2017-Robot-Code#1]
I'm looking at you, Clawdius/New_Claudius. Also, why is there a Shawntelle project despite the shooter being stolen for Discus Bot? Some deletion of projects or renaming is in order, perhaps?

**Comment by Alpine-Robotics** [comment]
I just deleted New_Claudius, which was a duplicate of Clawdius (the original "Forklift" project from 2015)
Shawntelle should be the unmodified code from that project year (2014?). At the end of the day yesterday we decided to start with a new blank iterative robot project call "DiscusBot" and add pieces from Clawdius and Shawntelle (and some new code) as needed next week. I was hoping to divide the group into separate teams to work on those subsystems in parallel (I'll also try to get my new laptop configured with Eclipse etc. so we can have 3 computers to code on)
BTW: We tried to rename and delete some of those projects using the Eclipse interface but couldn't check those changes in (git seemed to get confused). I finally had to use the gitbash tool and use command line directives to try to sort it all out. Since I never had those kind of problems in the past when the Eclipse workspace was also the parent git directory I'm thinking we might want to go back to that situation (rather than having a separate git directory that we import files from)

**Comment by JGelfand** [comment]
(For documentation purposes:)
Fixed in commit [a1031c15edaa26ebb95b9232d651dfacbc36b8df](https://github.com/FRCTeam159/2017-Robot-Code/commit/a1031c15edaa26ebb95b9232d651dfacbc36b8df)

**Comment by JGelfand** [comment]
Also: if the only problem with locating the repo in the default directory for Eclipse is difficulties when removing files, it's probably worth keeping them there for the convenience offered by, for example ease of changing workspaces. We shouldn't have to remove files from the repo very often, honestly. I'll probably put in a pull request soon removing the executables and the files I remember causing us trouble last year, and that should hopefully be the last of it.
